src/i_video.c

```diff
@@ -57,7 +57,7 @@
 #include "w_wad.h"
 #include "z_zone.h"
 
-#include "miniz.h"
+#include "spng.h"
```

Comment:
Hm, okay, but why? Does spng have any advantage over miniz in this regard?

Reply:
It has options, so by changing the compression level, I made it a little faster. It's not a big difference.

Reply:
How much is the difference in file size? I think the standard compression setting (probably also used by miniz) is 6. Could you do a small file size vs. compression time benchmark for values between 1 and 6?

Reply:
883x497 image
compression level 1 ~390 Kb
compression level 6 ~350 Kb
and it's 3x slower at level 6.